std/functional.d

```diff
@@ -537,11 +537,28 @@ alias equalTo = safeOp!"==";
 template reverseArgs(alias pred)
 {
     auto reverseArgs(Args...)(auto ref Args args)
-        if (is(typeof(pred(Reverse!args))))
+    if (is(typeof(pred(Reverse!args))))
     {
         return pred(Reverse!args);
     }
 }
+//
```

Comment:
did you mean to use `///` here?

Reply:
Yes. I copied the public tests from binaryReverseArgs as they could be kept.